types/clock.go

```diff
@@ -2,6 +2,7 @@ package types
 
 // ClockResponse main format for data coming out of clock api
 type ClockResponse struct {
+	ClockConfig
```

Comment:
Do we need to embed the Config here? Position needs to be embedded, but I'm not sure `Config` needs to be

Reply:
I thought it was best to return configs to everything. That way if a random client changes the widget's timezone, it would get updates on all clients.

Reply:
By embed I meant do we want the config data to be flattened out into the base of the Response Object?